### PR TITLE
Use toolset version from Arcade

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

**Comment by agocke** [comment]
@RikkiGibson Could you take a look at this when you get a chance?

File: .editorconfig [hunk 1]
```diff
@@ -37,7 +37,7 @@ indent_size = 2
 
 # Dotnet diagnostic settings
 [*.{cs,vb}]
-dotnet_diagnostic.xUnit2018.severity = suppress # "do not compare an object's exact type to the abstract class" is a valid assert, but very noisy right now
```
Comment (RikkiGibson) [inline .editorconfig]: I assume this fixes a bunch of spurious warnings in the build?

File: eng/Versions.props [hunk 2]
```diff
@@ -68,7 +68,6 @@
     <MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>14.3.25407-alpha</MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>
     <MicrosoftMetadataVisualizerVersion>1.0.0-beta1-63011-01</MicrosoftMetadataVisualizerVersion>
     <MicrosoftMSXMLVersion>8.0.0.0-alpha</MicrosoftMSXMLVersion>
-    <MicrosoftNetCompilersToolsetVersion>3.2.0-beta4-19312-15</MicrosoftNetCompilersToolsetVersion>
```
Comment (RikkiGibson) [inline eng/Versions.props]: So the point is, let's use the same compiler to build Roslyn as we give to everyone else via Arcade?

Comment (agocke) [reply eng/Versions.props]: Exactly.

File: .editorconfig (hunk 1 above)
Comment (agocke) [reply .editorconfig]: Yeah, this is reacting to a change where we renamed "suppress" in editorconfig to "none"

**Pull request merged**